rust/agama-server/src/web/common/issues.rs

```diff
@@ -66,6 +70,8 @@ pub enum IssuesServiceError {
     DBusName(#[from] zbus::names::Error),
     #[error("Could not send the event: {0}")]
     SendEvent(#[from] broadcast::error::SendError<Event>),
+    #[error("Issue conversion error")]
```

Comment:
NP: should we use `#[error(transparent)]`? In general, I am doing so when composing errors from other error type. But I am not sure if that is a common/good practice.